test/realm-fuzzer/fuzz_configurator.cpp

```diff
@@ -34,6 +35,7 @@ void FuzzConfigurator::setup_realm_config()
 {
     m_config.path = m_path;
     m_config.schema_version = 0;
+    m_config.scheduler = realm::util::Scheduler::make_dummy();
```

Comment:
yes, this makes sense, we should not need any scheduler while running the fuzzer since we are not testing any notification machinery...